src/main/modules/oidc/index.ts

```diff
@@ -59,6 +60,17 @@ export class OidcMiddleware {
           console.log('***** User login success');
           res.locals.isLoggedIn = true;
           req.locals.api = getCaseApi(req.session.user, req.locals.logger);
+          if (req.path.startsWith(C100_URL) || req.path.startsWith(DASHBOARD_URL)) {
+            const c100RebuildLdFlag: boolean =
+              req.session.c100RebuildLdFlag !== undefined
+                ? req.session.c100RebuildLdFlag
+                : (req.session.c100RebuildLdFlag = await getFeatureToggle().isC100reBuildEnabled());
+            if (c100RebuildLdFlag) {
+              return next();
+            } else {
+              return res.redirect(CITIZEN_HOME_URL);
+            }
+          }
           req.locals.C100Api = caseApi(req.session.user, req.locals.logger);
```

Comment:
We can remove this duplicate invocation ? caseApi is being invoked above block on line#62 as well

Reply:
Fixed it.